CONTRIB

```diff
@@ -169,3 +169,4 @@ Christian Zmasek <https://github.com/cmzmasek>
 Xiaoyu Zhuo <https://github.com/xzhuo>
 Harry Zuzan <iliketobicycle at domain yahoo.ca>
 Bertrand Néron < https://github.com/bneron>
+Oscar G. Garcia <https://github.com/oscarmaestre>
```

Comment:
This is meant to be alphabetical by surname - it looks like Bertrand's name is in the wrong place though.